INSTALL

```diff
@@ -5,54 +5,116 @@ Subsurface uses quite a few open source libraries and frameworks to do its
 job. The most important ones include libdivecomputer, Qt, libxml2, libxslt,
 libsqlite3, libzip, libgrantlee5 and libgit2.
 
-Below are instructions for building Subsurface under some popular
-Linux distributions, for building Subsurface using Homebrew on a Mac,
-and for cross-building Subsurface for Windows. The lack of a working
-package management system for Windows makes it really painful to build
-Subsurface natively under Windows, so we don't support that at all.
+Below are instructions for building Subsurface
+- on some popular Linux distributions,
+- MacOSX,
+- Windows (cross-building)
+- Android (cross-building)
+- iOS (cross-building)
 
-All of the prebuilt binaries that we provide (right now Windows, Mac,
-Ubuntu/Debian/LinuxMint, and openSUSE/Fedora) are built using our own
-custom "flavors" of libdivecomputer. You can get these from
 
-https://github.com/Subsurface-divelog/libdc.git
+Getting Subsurface source
+-------------------------
 
-If you have cloned subsurface it is easier,
-locate yourself in the subsurface repo, and do:
-git submoule init
-git submodule update
+You can get the sources to the latest development version from our git
+repository http://github.com/Subsurface-divelog/subsurface.git
 
-Those branches won't have a pretty history and will include ugly merges,
-but they should always allow a fast forward pull that tracks what we believe
-developers should build against.
+Remark, if you plan on making pull requests, the best way is to clone
+subsurface.git to your GitHub account, make a new branch (e.g. myPatches).
```

Comment:
I don't think this is the best way. At least I have subsurface-divelog/subsurface set as origin for my local repositories. Here I have a branch master which tracks origin/master. For coding I git checkout -b DESCRIPTIVE_TITLE . When done, I have my githup fork configured as a remote to which I push my branch as soon as it's ready for viewing (and finally create a PR). That way, I don't need to track origin/master remotely at GitHub.

In short, it's fine to clone the central repository, to create patches you'll need a fork, but that can be done later.

Reply:
Ok, I will update the text, to make it clear you can do a normal “git clone”. For pull requests I think we say  early the same, let me be sure:
- you have cloned subsurface to you github account
- you have cloned subsurface to your local disk
- you create a local branch, work, and when finished push upstream to your github form
- you make a pull request, on github as usual

did I get that right?

Reply:
My point was that I clone my local repository from github/subsurface-divelog (which thus becomes "origin") and not from GitHub/atdotde (which is just a remote like many other GitHub/username which I have to be able to check out branches of other developers). This allows me to easily track origin/master (on which my PR's should be based or rebased).